core/include/usModuleImport.h

```diff
@@ -51,15 +51,7 @@ US_END_NAMESPACE
  * \sa \ref MicroServices_StaticModules
  */
 #define US_INITIALIZE_STATIC_MODULE(_module_name)                          \
-  extern "C" void _us_import_module_initializer_ ## _module_name();        \
-  struct StaticModuleInitializer_ ## _module_name                          \
-  {                                                                        \
-    StaticModuleInitializer_ ## _module_name()                             \
-    {                                                                      \
-      _us_import_module_initializer_ ## _module_name();                    \
-    }                                                                      \
-  };                                                                       \
-  static StaticModuleInitializer_ ## _module_name _InitializeModule_ ## _module_name;
+  extern "C" void _us_import_module_initializer_ ## _module_name();
```

Comment:
I fixed the gcc build on Linux with [d337902] (https://github.com/jeffdiclemente/CppMicroServices/commit/d337902e1a427125efaad24bb59866fb6ee0af8c), however there are still unit test failures which will be resolved.